### ISSUE TITLE
Show all songs played (not just offline/cached songs) in the "songs" section 

### ISSUE THREAD

**Issue by dainvincible1** [anandnet/Harmony-Music#70]
There is a separate section for cached songs. So, it'd be nice if the song section shows all the songs i have played not just the offline or cached songs.
Sometimes i play song for sometime then close the app, it'll be better if those songs are also shown in the "songs" section

**Comment by siggi1984** [comment]
I think it's not bad that the Songs section shows the offline songs. I personally would like to see all played songs in the Recently Played section and not only the last 20 songs.

**Comment by dainvincible1** [comment]
> I think it's not bad that the Songs section shows the offline songs. I personally would like to see all played songs in the Recently Played section and not only the last 20 songs.

yes ofc its not bad...but it'll be good if it shows all the songs played including the offline as well as not offline cause as you said recently played songs show only 20 songs, it'd be good if all songs are shown somewhere
